constants.py

```diff
@@ -84,19 +84,22 @@ def error_string(error, logname, with_hd):
 
 # filesystems and partitions (sizes in MB):
 boot_size = 512
-root_mbr_size_old = 4096
-root_mbr_size = 18432
-root_gpt_size_old = 3584
-root_gpt_size = 17920
-root_size_old = max(root_mbr_size_old, root_gpt_size_old)  # used for free space calculations
-root_size = max(root_mbr_size, root_gpt_size)  # used for free space calculations
-backup_size_old = 4096
+root_size = 18432
 backup_size = 18432
 swap_file_size = 512
 swap_size = 1024
 logs_size = 4096
 logs_free_space = 20
 
+install_size = logs_size + backup_size + root_size + boot_size + swap_size
+
+# Number of sectors consumed by the GPT on a disk with 512 byte sectors.
+# 1 sector for MBR, 1 sector for GPT header + 32 sectors for partition entries.
+gpt_header_512_sectors = 34
+# Number of sectors consumed by the backup GPT on a disk with 512 byte sectors.
+# 1 sector for GPT header + 32 sectors for partition entries.
+backup_gpt_header_512_sectors = gpt_header_512_sectors - 1
```

Comment:
These should be functions depending on sector size, not assuming the sector size is always 512.